app/src/datasets/chorus_dt/places_resolution.py

```diff
@@ -23,18 +23,10 @@ def compute_distances(data):
     data["distance_1"] = np.NaN
     data["distance_2"] = np.NaN
 
-    pairs = [("lieu_depart", "lieu_etape"), ("lieu_etape", "lieu_arrivee"), ("lieu_depart", "lieu_arrivee")]
-
-    for distance_i, (column_0, column_1) in enumerate(pairs):
-        data["distance_%s" % distance_i] = compute_distance_between_points(
-            data["%s_lon" % column_0], data["%s_lat" % column_0], data["%s_lon" % column_1], data["%s_lat" % column_1]
-        )
-    data["distance_0"].fillna(0, inplace=True)
-    data["distance_1"].fillna(0, inplace=True)
-    data["distance_2"].fillna(0, inplace=True)
+    data["distance"] = compute_distance_between_points(
+        data["lieu_depart_lon"], data["lieu_depart_lat"], data["lieu_arrivee_lon"], data["lieu_arrivee_lat"]
+    )
 
```

Comment:
do we need to add 
```suggestion
data["distance"].fillna(0, inplace=True)
```
?

Reply:
done